### ISSUE TITLE
You forgot to include screen.php

### ISSUE THREAD

**Issue by danieleftodi** [hyyan/woo-poly-integration#435]
Solution:

require_once(ABSPATH . 'wp-admin/includes/screen.php'); 

add it to:

/wp-content/plugins/woo-poly-integration/src/Hyyan/WPI/Emails.php

Three days ago you update the plugin ... and our Gift Card implementation with Yith Gift Cards couldn't send out any e-mails ... 

as on line 270 you where refering to an undeclared function ...

$screen = get_current_screen();

Found the solution online, and allied it to screen.php ... and it works again....

**Comment by Jon007** [comment]
Hi @danieleftodi  in what circumstance is this, on which screen?

in admin mode get_current_screen() is normally defined, 
the correct fix may probably be to test function_exists( 'get_current_screen') rather than requiring it since on the orders screen where we are sending the emails it is defined..

are you saying that YITH Gift Cards has an admin screen which sends emails and where get_current_screen is not defined?

**Comment by danieleftodi** [comment]
In this "Screen" ... when clicking "Send now" 

<img width="1792" alt="Screenshot 2019-06-27 at 22 08 52" src="https://user-images.githubusercontent.com/35347921/60297456-8162fb00-9928-11e9-80dd-12bbcbbd8f02.png">


Also, I don't know, but I'm guessing when the system is automatically sending out the "Gift Card"  order... when the order is marked as "Payed" in WooCommerce ... it is using the same email function.

**Comment by Jon007** [comment]
@danieleftodi  
>the WP-Security plugin
I don't know, I have not tried this one..

What this piece of code does is, for any email using standard woocommerce footer text, from address and from name, if we are in admin mode, only attempt to translate these values if the current screen is the order detail screen...    .. the reason for doing this is that we can only reliably tell the language when in the current order screen by detecting the language of the global $post object. 

These filters are generic - for example also called from the password reminder screen - so do not include any information about the current order as this may not be an order email.

 Basically there are several possible solutions and I am trying to understand which is the best one:

1. simply exit and ignore if get_current_screen is not defined (in admin mode, only translate if on the expected order detail screen viewing a single order)
2. ignore current screen and get more clever about language detection (check if there is a global post object and it has a language, if not check if Polylang filtered language admin view is set and use the filtered language..)
3. ..
